v3/src/components/graph/models/graph-layout.ts

```diff
@@ -63,18 +63,18 @@
       // the plot. We tried work arounds to get gridlines that were _not_ part of the axis element with the result
       // that the gridlines got out of synch with axis tick marks during drag. So we have this inelegant solution
       // that shouldn't affect the top and right axes when we get them but
-      // todo: check to make sure this still works with top and right axes
-      const newBounds = {
-        left: bounds.left,
-        top: place === 'bottom' ?
-          Math.min(bounds.top, this.axisLength('left')) : bounds.top,
-        width: place === 'left' ?
-          Math.min(bounds.width, this.graphWidth - this.axisLength('bottom')) : bounds.width,
-        height: place === 'bottom' ?
-          Math.min(bounds.height, this.graphHeight - this.axisLength('left') - this.legendHeight) :
-          place === 'left' ?
-            Math.min(bounds.height, this.graphHeight - this.legendHeight) : bounds.height
+
+      // given state of the graph, we may need to adjust the drop areas' bounds
+      const newBounds = bounds
+
+      if (place === "bottom"){
+        newBounds.top = this.plotHeight
       }
+
+      if (place === "left" && bounds.width > this.plotWidth){
+         newBounds.width -= this.plotWidth
+      }
+
       this.axisBounds.set(place, newBounds)
```

Comment:
I entered a reply about the importance of computations for height, but the reply disappeared Along with the conversation thread. :-(